### DISCUSSION TITLE
Error when using Translation Provider bundle with DataHub Webhook bundle installed

### DISCUSSION THREAD

**Discussion by Maksonis** [pimcore/pimcore#16021] [Q&A / Support] [answered]
Hi, 

I have installed DataHub Webhook bundle and when I try to run one of translation provider bundle commands getting this:

In Service.php line 489: Pimcore\Model\Element\Service::getElementType(): Argument #1 ($element) mus t be of type Pimcore\Model\Element\ElementInterface, Pimcore\TranslationsProviderInterfaceBundle\Entity\Job given, called in /var/www/html/vendor/pimcore/data-hub-webhooks/src/EventListener/WorkflowEventListener.php on line 51 translations-provider-interface-bundle:jobs [-c|--create-job-from-changed-items] [-o|--process-open-jobs] [-s|--process-submitted-jobs] [-r|--process-received-jobs] [--delete-jobs] [--min-translation-items MIN-TRANSLATION-ITEMS] [--check-redeliveries] [-f|--reset-failed]

Is this a bug ?

**Comment by dvesh3** [comment]
@Maksonis this is a known issue and already reported on datahub/webhooks to handle it. Thanks!

**Reply by dvesh3** [reply, marked as answer]
Fixed with `datahub/webhooks:v1.0.1`.
